yarn-project/barretenberg.js/src/crypto/grumpkin/index.ts

```diff
@@ -37,11 +57,20 @@ export class Grumpkin {
     return result;
   }
 
+  /**
+   * Gets a random field element.
+   * @returns Random field element.
+   */
   public getRandomFr() {
     this.wasm.call('ecc_grumpkin__get_random_scalar_mod_circuit_modulus', 0);
     return Buffer.from(this.wasm.getMemorySlice(0, 32));
   }
 
+  /**
+   * Converts a 512 bytes long buffer to a field.
```

Comment:
```suggestion
   * Converts a 512 bits long buffer to a field.
```

Assuming uint512_t in cpp follows the same convention as uint8_t

Reply:
Good catch. Thanks